components/list/list.js

```diff
@@ -47,7 +62,9 @@ class List extends PageableMixin(SelectionMixin(LitElement)) {
 			 * @type {'all'|'between'|'none'}
 			 * @default "all"
 			 */
-			separators: { type: String, reflect: true }
+			separators: { type: String, reflect: true },
+			_breakpoint: { type: Number, reflect: true },
+			_slimColor: { type: Boolean, reflect: true, attribute: '_slim-color' }
```

Comment:
Let me know if there's a better/preferred way to do this than internal properties

Reply:
If we want to avoid sprouting these private properties on the `host` then I think we'd have to render a class on the `div`, but then elements in the other slots would not inherit the variables. Will any of these slots need them?

Reply:
Hmm it's possible we'd want `_slimColor` for the `controls` in order to leave correct spacing at slim width. Currently I think it only leaves correct spacing at larger width.

Reply:
In that case we'd either need to keep sprouting on the `host` like this, or introduce a new internal wrapper `div`. Not sure if introducing a new `div` will break anything.

Reply:
Another way to avoid sprouting would be to set the variables through JavaScript using `this.style.setProperty('--d2l-list-color-width', '3px');`. I'm not sure I like that any better though... I'd probably just sprout if it came to that.

Reply:
Quickly adding a wrapper div does break almost all the visual diffs but they're just all 1px taller, which isn't terrible but also scares me a bit